src/components/Calculator.js

```diff
@@ -3,6 +3,7 @@ import CalcWrapper from './CalcWrapper';
 import CalcScreen from './CalcScreen';
 import ButtonsPad from './ButtonsPad';
 import Button from './Button';
+import CalcProvider from '../context/CalcContext';
```

Comment:
- [ ] As said above 🙃  The `Calculator` component should import the `calculate.js` file from the logic directory:

```javascript
// ./src/components/Calculator.js

import React, { Component } from 'react';
import calculate from ../logic/calculator;
```

   * `operate.js` is already being used by `calculate.js`, so there's no need to import it

Reply:
Thanks Mr. @BenMukebo  for the great feedback. I have already implemented the requested changes.